app/views/lead_providers/partnership_csv_uploads/new.html.erb

```diff
@@ -0,0 +1,35 @@
+<% content_for :title, "Upload partnership csv" %>
+<% content_for :before_content, govuk_back_link( text: "Back", href: '#') %>
```

Comment:
I think the previous page exists already?

Reply:
Yep, will update